packages/fast-tooling-react/src/viewer/select-device/select-device.style.ts

```diff
@@ -39,6 +39,7 @@ const styles: ComponentStyles<
         },
         selectDevice_select: {
             ...applySelectInputStyles(),
+            paddingRight: "15px",
```

Comment:
not sure... are we are supporting rtl here?

Reply:
the location of the dropdown arrow shouldn't be switching on LTR, but regardless the tooling components do not take into account RTL as they are.